libcrux_crypto/src/crypto.rs

```diff
@@ -0,0 +1,578 @@
+use std::sync::{Mutex, MutexGuard};
+
+use libcrux::drbg::{Drbg, RngCore};
+use libcrux::hpke::{self, HPKEConfig};
+use openmls_traits::crypto::OpenMlsCrypto;
+use openmls_traits::types::{
+    AeadType, Ciphersuite, CryptoError, ExporterSecret, HashType, HpkeAeadType, HpkeCiphertext,
+    HpkeConfig, HpkeKdfType, HpkeKemType, HpkeKeyPair, KemOutput, SignatureScheme,
+};
+
+use rand::CryptoRng;
+use tls_codec::SecretVLBytes;
+
+const MAX_DATA_LEN: usize = 0x10000000;
```

Comment:
Where is this number coming from? Could you add a code comment?

Reply:
I was sure I got that from the called code, but I couldn't find it. Reading the AEAD RFC told me that this number is different for each ciphersuite, so I just removed it and let libcrux take care of it. The errors will be less specific but such is life.